docs/advanced/opt.md

```diff
@@ -24,6 +24,10 @@ The [BFGS method](https://en.wikipedia.org/wiki/Broyden%E2%80%93Fletcher%E2%80%9
 
 In ABACUS, we implemented the BFGS method for doing fixed-cell structural relaxation.
 
+### BFGS_TRAD method
+
+The BFGS_TRAD method is an algorithm implemented in ABACUS, referencing the BFGS method from ASE. The previous BFGS method in ABACUS did not perform well for some tests and the BFGS_TRAD method now produces results that are consistant with the BFGS method in ASE. In cases where the previous BFGS method could not converge within a limited number of steps, the BFGS_TRAD method can successfully converge.
```

Comment:
This method is not comes from ASE

Reply:
@19hello This method may be the BFGS used by ASE, but not from ASE. You need to specify the main algorithm this BFGS and its difference with old BFGS.

Reply:
I have modified the introduction of BFGS in the document.